File: lap.cpp
#include <iostream>
using namespace std;

float f(float x){
	return 5*pow(x,3)-20*x-3; //khai bao ham f(x)
}
float g(float x){
	return (5*pow(x,3)+3)/20; //ham tuong duong f(x)
}
float lap(float a, float b){
	float exp = 0.0001;
	float x = a;
	float y;
	do{
		y = x;
		x = g(x);
	}while(abs(x-y)>exp);
	return x;
}
int main(){
	float a,b;
	cout<<"Nhap 1 khoang ly nghiem a,b: ";
	cin>>a>>b;
	cout<<"Nghiem gan dung: "<<lap(a,b);
}

File: lap.py
def g(x):
    # Định nghĩa hàm g(x) của phương trình
    return (-x**3+3*x**2+24*x-5)/24

def lap(x, epsilon):
    # Tìm nghiệm gần đúng của phương trình
    y=0
    while abs(x - y) > epsilon:       
        y = x
        x = g(x)
        #print(x,y)
    return x

print(lap(-1.5,0.0128))